plugin-metadata/plugin-security.policy

```diff
@@ -6,4 +6,5 @@ grant {
   permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
   permission java.util.PropertyPermission "*", "read,write";
   permission java.net.SocketPermission "*", "accept, resolve, connect";
+  permission java.net.NetPermission "getProxySelector";
```

Comment:
why do we need it?

Reply:
es60x needs this one to create HLC

Reply:
oh, ok. So, I'd like you to do one more improvement - please let's:
* copy the `plugin-metadata` folder to each es module.
* leave the permission only for es60x
* remove `plugin-metadata` from root folder
* adapt `resolvePluginDescriptorTemplate` task